script.js

```diff
@@ -0,0 +1,66 @@
+let word = 'APPLE'
```

Comment:
I think we need to change approach here and instead of working with the whole word, you should split the word into individual letters
I mentioned a JS method called split() which will do this
If you do:
`word.split("")`
This should generate for you an array with the separate letters for word
So it should look like:
`[a,p,p,l,e]`
And from here on you can work with this array instead of the whole word